demos/tests/cases.py

```diff
@@ -391,6 +391,22 @@ def single_option_cases(key, *args):
             }),
         ],
     )),
+
+    PythonDemo(subdirectory='text_detection_demo_opencv', test_cases=combine_cases(
```

Comment:
Note that `PythonDemo` now has a `device_keys` argument. Since this demo doesn't have any device keys, you should set it to `[]`.

Reply:
done

Reply:
You missed a comma.